llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -28729,7 +28731,7 @@ static SDValue GenerateFixedLengthSVETBL(SDValue Op, SDValue Op1, SDValue Op2,
   unsigned BitsPerElt = VTOp1.getVectorElementType().getSizeInBits();
   unsigned IndexLen = MinSVESize / BitsPerElt;
   unsigned ElementsPerVectorReg = VTOp1.getVectorNumElements();
-  uint64_t MaxOffset = APInt(BitsPerElt, -1, false).getZExtValue();
+  uint64_t MaxOffset = APInt(BitsPerElt, -1, true).getZExtValue();
```

Comment:
Seems convoluted - `APInt::getAllOnes(BitsPerElt).getZExtValue()` or just do a `~0ULL >> (64 - BitsPerElt)`?

Reply:
After poking around in MathExtras a bit, I replaced this with `maxUIntN(BitsPerElt)`.